Documentation/gettingstarted/k8s-install-openshift-okd.rst

```diff
@@ -118,54 +118,38 @@ Next, generate OpenShift manifests:
 
    openshift-install create manifests --dir "${CLUSTER_NAME}"
 
-Now, define ``cilium`` namespace:
+Next, obtain Cilium manifest from ``cilium/cilium-olm`` repository and copy to ``${CLUSTER_NAME}/manifests``:
 
-.. code:: bash
+.. parsed-literal::
 
-   cat << EOF > "${CLUSTER_NAME}/manifests/cluster-network-03-cilium-namespace.yaml"
-   apiVersion: v1
-   kind: Namespace
-   metadata:
-     name: cilium
-     annotations:
-       # node selector is required to make cilium-operator run on control plane nodes
-       openshift.io/node-selector: ""
-     labels:
-       name: cilium
-       # run level sets priority for Cilium to be deployed prior to other components
-       openshift.io/run-level: "0"
-       # enable cluster logging for Cilium namespace
-       openshift.io/cluster-logging: "true"
-       # enable cluster monitoring for Cilium namespace
-       openshift.io/cluster-monitoring: "true"
-   EOF
+   cilium_olm_rev=master
 
-.. include:: k8s-install-download-release.rst
+   curl --silent --location --fail --show-error "https://github.com/cilium/cilium-olm/archive/${cilium_olm_rev}.tar.gz" --output /tmp/cilium-olm.tgz
+   tar -C /tmp -xf /tmp/cilium-olm.tgz
 
-Next, render Cilium manifest:
+   cp -a "/tmp/cilium-olm-${cilium_olm_rev}/manifests/cilium.v|RELEASE|/" "${CLUSTER_NAME}/manifests"
```

Comment:
Perhaps `|RELEASE|` is just not working here because it's not surrounded by spaces? In all places where we have e.g. `|CHART_RELEASE|`, it is surrounded by spaces...

Reply:
...somewhat weirdly `\  |RELEASE|\ ` seems to work!

Reply:
@errordeveloper I gave these instructions a shot and with the following `cp` command I end up having a `cilium.v1.9.5` directory inside the `${CLUSTER_NAME}/manifests` folder. Not sure if that is correct or not (I never encountered a subfolder inside the `manifests` dir).

Reply:
Good catch, I will have another look.

Reply:
@ungureanuvladvictor I think it's very likely that you might have missed the leading slash in the first argument to `cp`, e.g. because you might have used shell auto-completion. I think that's a good enough reason to change how this is step is documented and make it less error-prone.

Reply:
xref https://github.com/cilium/cilium-olm/issues/32